### PR TITLE
Add echelon to actor data and roll data

### LINKED ISSUE #106: Include character echelon as getter on BaseActorModel and in roll data
The derived value of a character's echelon - e.g. 1 for a level 1-3 character - should be available both in the system properties as well as roll data. This should be available for both characters and npcs.

### PR DESCRIPTION
**Pull request by cdverrett94**
Closes #106 

- I had to add level to `NPCModel`. 
- Added a level getter so that I could use that instead of actor type checking in the echelon getter. 
- Added echelon to roll data

### CODE REVIEW

File: src/module/data/actor/base.mjs [hunk 1]
```diff
@@ -72,6 +72,22 @@ export default class BaseActorModel extends foundry.abstract.TypeDataModel {
...
+   */
+  get level() {
+    return 1;
+  }
+  
+  /**
+   * The actor's echelon based on their current level
+   */
+  get echelon() {
+    return Object.entries(ds.CONFIG.echelons).reduce((acc, current) => {
```
Comment (JPMeehan) [inline src/module/data/actor/base.mjs:86]: You can destructure `current` to something like `entry[key, value]` so you can just have `value.threshold` and `Number(key)`

File: src/module/data/actor/base.mjs [hunk 2]
```diff
@@ -72,6 +72,22 @@ export default class BaseActorModel extends foundry.abstract.TypeDataModel {
...
+      return this.level >= current[1].threshold? Number(current[0]) : acc;
```
Comment (JPMeehan) [inline src/module/data/actor/base.mjs:87]:
```suggestion
      return this.level >= current[1].threshold ? Number(current[0]) : acc;
```

**Review by JPMeehan — APPROVED** [review]
I waffled a bit on whether the echelon stuff should go in `modifyRollData` - merge/rebase from main if you don't see it, I added it in the Potency PR that just got merged - but I think it's a sufficiently universal concept that every actor subtype should probably provide it.

File: src/module/data/actor/base.mjs (hunk 1 above)
Comment (cdverrett94) [reply src/module/data/actor/base.mjs]: Thanks. Didn't think to destructure there

**Comment by cdverrett94** [comment]
Updated per your comments. Moved the echelon roll data to `modifyRollData`.

**Pull request merged**